internal/app/carrier/repository_test.go

```diff
@@ -0,0 +1,77 @@
+package carrier
+
+import (
+	"context"
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"parcel-service/internal/app/model"
+	"testing"
+	"time"
+)
+
+func TestRepository_UpdateCarrierRequest(t *testing.T) {
```

Comment:
need to write test for each query failed inside transaction and also transaction failed

Reply:
all changes are done, Please look over it.